tools/python/run_CIs_for_external_pr.py

```diff
@@ -25,9 +25,9 @@ def parse_args():
     return args
 
 
-def run_gh_pr_command(command: typing.List[str]):
+def run_gh_pr_command(command: typing.List[str], check=True):
     try:
-        return subprocess.run(["gh", "pr", *command], capture_output=True, text=True, check=True)
+        return subprocess.run(["gh", "pr", *command], capture_output=True, text=True, check=check)
```

Comment:
Out of interest why is check=False when checking output about passed pipelines? Do we expect that to fail for a valid PR that is currently open?

Reply:
`gh pr checks` exits with non-zero exit code when failures in pipeline exist